utils/node_local_test.py

```diff
@@ -1300,6 +1327,10 @@ def log_test(conf,
 
     compress_file(filename)
 
+    if conf.max_log_size and fstat.st_size > conf.max_log_size:
```

Comment:
Very minor suggestion. Perhaps also include the configured max_log_size in the output to make it easier to interpret the failure without having to look through the rest of the log and notice the invocation including the --max-log-size command line argument?

Reply:
Typically when this fails it's because of a bug in the PR and the log is much bigger so I don't think that would help much.

Reply:
fair enough, thanks for considering it

Reply:
Done.